Source/Templates/GtkSharp.Template.CSharp/content/GtkSharp.CompositeWidget.CSharp/Gtk_Composite_Widget.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Gtk;
+
+namespace GtkNamespace
+{
+    [Template("Gtk_Composite_Widget.glade")]
+    [TypeName(nameof(Gtk_Composite_Widget))]
+    public class Gtk_Composite_Widget : Bin
```

Comment:
C# naming scheme please: `GtkCompositeWidget`

Reply:
Done

Reply:
You have missed several places where there is `Gtk_Composite_Widget`.

Reply:
Okay, updated all occurrences